Failures on 2 unittests

#### Brief summary of issue / Description of requested feature:

I get a repeated error when running 'evennia test evennia' even from a fresh install.

#### Steps to reproduce the issue / Reasons for adding feature:

1. Run `evennia test evennia` (the game directory could have been created just previously, without any changes to the default configuration).

#### Error output / Expected result of feature

```
======================================================================
FAIL: test_cmdgametime (evennia.contrib.tests.TestExtendedRoom)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\vincent\src\mud\evennia\evennia\contrib\tests.py", line 219, in test_cmdgametime
    self.call(extended_room.CmdGameTime(), "", "It's a spring day, in the evening.")
  File "c:\users\vincent\src\mud\evennia\evennia\commands\default\tests.py", line 83, in call
    raise AssertionError(retval)
AssertionError: 
==============================Wanted message==================================
It's a spring day, in the evening.
==============================Returned message================================
It's a spring day, in the afternoon.
==============================================================================

======================================================================
FAIL: test_return_appearance (evennia.contrib.tests.TestExtendedRoom)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\vincent\src\mud\evennia\evennia\contrib\tests.py", line 198, in test_return_appearance
    self.assertEqual("evening", self.room1.ndb.last_timeslot)
AssertionError: 'evening' != 'afternoon'

----------------------------------------------------------------------
Ran 193 tests in 31.328s

FAILED (failures=2)
```

Unfortunately it still fails... there is even an additional test that fails now, `FAIL: test_return_appearance (evennia.contrib.tests.TestExtendedRoom)`.  Here's the traceback, if it helps you in any way:

```
FAIL: test_return_appearance (evennia.contrib.tests.TestExtendedRoom)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\vincent\evennia\evennia\contrib\tests.py", line 196, in test_re
turn_appearance
    self.room1.return_appearance(self.char1))
AssertionError: '|cRoom(#1)|n\nA spring description.\n|wExits:|n out(#3)\n|wYou
see:|n Obj(#4), Obj2(#5), Char2(#7)' != '|cRoom(#1)|n\nroom_desc\n|wExits:|n out
(#3)\n|wYou see:|n Obj(#4), Obj2(#5), Char2(#7)'
```

--- a/evennia/contrib/tests.py
+++ b/evennia/contrib/tests.py
@@ -4,6 +4,7 @@ Testing suite for contrib folder
 
 """
 
+import datetime
 from evennia.commands.default.tests import CommandTest
 from evennia.utils.test_resources import EvenniaTest
 from mock import Mock, patch
@@ -173,6 +174,16 @@ from evennia.contrib import extended_room
 from evennia import gametime
 from evennia.objects.objects import DefaultRoom
 
+class ForceUTCDatetime(datetime.datetime):
+
+    """Force UTC datetime."""
+
+    @classmethod
+    def fromtimestamp(cls, timestamp):
+        """Force fromtimestamp to run with naive datetimes."""
+        return datetime.datetime.utcfromtimestamp(timestamp)
+
+@patch('evennia.contrib.extended_room.datetime.datetime', ForceUTCDatetime)
 class TestExtendedRoom(CommandTest):
     room_typeclass = extended_room.ExtendedRoom
     DETAIL_DESC = "A test detail."

